src/webserver.cpp

```diff
@@ -465,27 +464,34 @@ MHD_Result webserver::post_iterator(void *cls, enum MHD_ValueKind kind,
     struct details::modded_request* mr = (struct details::modded_request*) cls;
 
     try {
-        if (filename == nullptr || mr->ws->file_upload_target != FILE_UPLOAD_DISK_ONLY) {
+        if (filename == nullptr ||
+            mr->ws->file_upload_target != FILE_UPLOAD_DISK_ONLY) {
             mr->dhr->set_arg(key, mr->dhr->get_arg(key) + std::string(data, size));
         }
 
-        if (filename && *filename != '\0' && mr->ws->file_upload_target != FILE_UPLOAD_MEMORY_ONLY) {
+        if (filename &&
+            *filename != '\0' &&
+            mr->ws->file_upload_target != FILE_UPLOAD_MEMORY_ONLY) {
```

Comment:
nit: The length of this line isn't long enough to justify the break. You can keep it on one line for easier readability